src/task4/routers/Users.ts

```diff
@@ -0,0 +1,78 @@
+
+import {
+  UserType,
+  UserSchema
+} from "../typings";
+import {
+  ValidatedRequest
+} from "express-joi-validation";
+import {
+  Response,
+  Request,
+  NextFunction
+} from "express";
+import UserService from "../services/UserService";
+
+const userService = new UserService();
+
+export const getUsers = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
please review the repo: https://github.com/santiq/bulletproof-nodejs/tree/master/src/api/routes

or 

https://github.com/santiq/bulletproof-nodejs/blob/master/src/loaders/express.ts

Your structure is similar to the recommended one, but with some differences (e.g. it should be routes not routers ;) ).  Analyze how loaders/routes are registered